mypy.ini

```diff
@@ -11,6 +11,7 @@ warn_unused_configs = true
 no_implicit_optional = true
 ignore_missing_imports = true
 namespace_packages = true
+plugins = envier.mypy
```

Comment:
I have no idea what this does 😅

Reply:
Adds a typing plugin to mypy to type-check the variables associated with environment variables. E.g. if config.interval is an `int` and we try to use it where a `str` is expected, then mypy will give us an error and we can fix the potential bug before it gets in the wild 🙂 .